src/dao/DAO.js

```diff
@@ -190,26 +191,8 @@ class DAO<TEntity: { id: EntityID }, TEntityMutator> extends BaseDAO<
   }
 
   fetchCustom<TResult>(queryOptions?: QueryOptions): LoadObject<TResult> {
```

Comment:
I feel like you don't really need this function. Anything that is custom should just call `__fetchCustom` since you'll need to do some sort of string formatting anyways.